composer/loggers/tensorboard_logger.py

```diff
@@ -31,19 +32,19 @@ class TensorboardLogger(LoggerDestination):
             will be saved. This is also the value that should be specified when starting
             a tensorboard server. e.g. `tensorboard --logdir={log_dir}`. If not specified
             `./tensorboard_logs` will be used.
-        flush_interval (int, optional): How frequently by batch to flush the log to a file.
-            For example, a flush interval of 10 means the log will be flushed to a file
-            every 10 batches. The logs will also be automatically flushed at the start and
-            end of every evaluation phase (`EVENT.EVAL_START` and `EVENT.EVAL_END` ),
-            the end of every epoch (`EVENT.EPOCH_END`), and the end of training
-            (`EVENT.FIT_END`). Default: ``100``.
+        flush_secs (int, optional): How frequently in seconds to flush the log to a file
```

Comment:
discussed offline, but could we put the change to make the flush interval in seconds to a seperate pr? would like to cherry-pick the fix into 0.8.1, but changing the interval to seconds would be a breaking change.

Reply:
https://github.com/mosaicml/composer/pull/1283